src/cargo/core/profiles.rs

```diff
@@ -561,6 +561,7 @@ fn merge_profile(profile: &mut Profile, toml: &TomlProfile) {
         profile.panic = match panic.as_str() {
             "unwind" => PanicStrategy::Unwind,
             "abort" => PanicStrategy::Abort,
+            "immediate-abort" => PanicStrategy::ImmediateAbort,
             // This should be validated in TomlProfile::validate
             _ => panic!("Unexpected panic setting `{}`", panic),
```

Comment:
The validation in there needs to be updated

Reply:
I'm afraid I'm lost. What validation? I can't find any validation of the panic setting or a `TomlProfile::validate`.

Reply:
https://github.com/rust-lang/cargo/blob/56bdf49f7b7102d90b3d72f3452544d5ca1f76d0/src/cargo/util/toml/mod.rs?plain=1#L2552-L2560

Reply:
Outdated comment always 🙇🏾‍♂️